packages/sdk/src/client/backend.ts

```diff
@@ -73,36 +79,53 @@ export class MetromApiClient {
             (await response.json()) as FetchCampaignsResponse;
 
         return {
-            campaigns: rawCampaignsResponse.campaigns.map((rawCampaign) => ({
-                ...rawCampaign,
-                pool: {
-                    ...rawCampaign.pool,
-                    amm: rawCampaign.pool.amm as SupportedAmm,
-                    token0: {
-                        ...rawCampaign.pool.token0,
-                        chainId: this.chain,
-                    },
-                    token1: {
-                        ...rawCampaign.pool.token1,
-                        chainId: this.chain,
+            campaigns: rawCampaignsResponse.campaigns.map((rawCampaign) => {
+                const rewards: Rewards = Object.assign([], { valueUsd: 0 });
+                for (const rawReward of rawCampaign.rewards) {
+                    let valueUsd = null;
+                    if (rewards.valueUsd !== null && rawReward.priceUsd) {
+                        valueUsd =
+                            Number(
+                                (rawReward.amount / 10n) ^
```

Comment:
Here we should use  `Math.pow`, no? Isn't `^` a bitwise XOR operator?

Reply:
Fuuuuuck you're right. Addressed in 4c1015ac3af4accf0ab00b2c92414a569970e17e